src/App.jsx

```diff
@@ -29,18 +29,24 @@ const App = () => {
     EOSIconsList[option].forEach((icon) => {
       const isFilledAvailable = "hasOutlined" in icon && icon.hasOutlined;
       const isFilledSelected = theme === "Filled";
+      const iconName = icon.name;
       if (theme === "All") {
         const nameFilledIcon = `Eos_${icon.name}_FILLED`.toUpperCase();
         const nameOutlinedIcon = `Eos_${icon.name}_OUTLINED`.toUpperCase();
         const EOSFilledIcon = EOSIcons[nameFilledIcon];
         const EOSOutlinedIcon = EOSIcons[nameOutlinedIcon];
         if (EOSFilledIcon) {
-          icons.push({ EOSReactIcon: EOSFilledIcon, name: nameFilledIcon });
+          icons.push({
+            EOSReactIcon: EOSFilledIcon,
+            name: nameFilledIcon,
+            showName: iconName,
```

Comment:
It can be better named as `displayName`?

Reply:
sure I will work on it